frontend/src/common/components/searchparamrouter/SearchParamRouter.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { IDPS } from "../../types/idp";
+import { useAuth } from "react-oidc-context";
+import { InitialLandingPage } from "../../../features/homePage/InitialLandingPage";
+
+const SearchParamRouter = () => {
```

Comment:
maybe call it `HomeRouter`? I dunno - `SearchParamRouter` is technically accurate...

Reply:
sounds good to me 👍